### DISCUSSION TITLE
Suggest a Book for the Community (#beat0quote)

### DISCUSSION THREAD

**Discussion by trainfarb** [hu3agency/ep0_daoTeaching#23] [#b0mission]
Hey 👋

ourLongPlay is hu3’s open-source game where people help each other create mixtapeMythologies. This is the first move every time a person starts a round in the game 🃏…planting a seed in people’s imaginations with a book quote and a song.

📚#beat0quote: The first move in ourLongPlay is sharing a book quote that’s meaningful to you. Music is the foundation of this game, so you’ll also choose a song to go with the book quote you pick. This is a simple way for anyone to open-source the tone of the language used in any decentralized subculture.

Add a comment below that includes:
1. A book quote that’s meaningful to you (book title, author, quote) 
2. Share a song you like (song title, artist)
3. Introduce yourself to the gang (stuff you like, favorite super hero, and any social media usernames where you want people to follow you)

Thanks for playing!

**Comment by Jamesdimitrov** [comment]
https://open.spotify.com/track/1rHQVcUIkpeHrzj6z74Emy?si=_FafrwOpThuvDRqDwpEJ8A&utm_source=copy-link

☝️Song: Microdosio: Cubensis

if those dried-up little scraps of fungus taught me anything, it is that there are other stranger forms of consciousness available to us, and, whatever they mean, their very existence, to quote William James, "forbids[s] a premature closing of our accounts with reality


Quote Source: Michael Pollan, How to Change Your Mind: What the New Science of Psychedelics Teaches Us About Consciousness, Dying, Addiction, Depression, and Transcendence

**Reply by trainfarb** [reply]
Thanks! Made a discussion dedicated to this book [here](https://github.com/hu3agency/LP0_polymathProcess/discussions/35#discussion-3768833), so you can add book quotes as often as you'd like. This song was added to the EP0_LessonPlan_#beat0quote](https://open.spotify.com/playlist/3Lhcxov4WZ1wv1VYABf2uy?si=e585e9bdff12470f) playlist.
